apistrap/utils.py

```diff
@@ -34,3 +36,33 @@ def snake_to_camel(value):
     """
     result = "".join(x.capitalize() or "_" for x in value.split("_"))
     return result[0].lower() + result[1:]
+
+
+def get_function_perspective_globals(function, level=0, max_level=None):
```

Comment:
missing param/return typehints & brief doc

maybe rename to `_get_function_perspective_globals` and leave without docs